navigation.rb

```diff
@@ -0,0 +1,16 @@
+And /^I click on app_launcher icon$/ do
+  App.power_nap(App::MIN_SLEEP)
+  on(NavigationPage).input_tag('app_launcher')
+  App.power_nap(App::MIN_SLEEP)
+end
+
+And /^I click on "([^"]*)"$/ do |console_type|
+  sleep 5
```

Comment:
Use power nap method instead of sleep and comment the reason when the power nap method is been added